docs/pages/world-explorer.mdx

```diff
@@ -0,0 +1,163 @@
+import { Tabs, Tab } from "nextra/components";
+import { CollapseCode } from "../components/CollapseCode";
+
+# World Explorer
+
+World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+It is available in the MUD TypeScript templates starting with version 2.2.
+
+## Usage
+
+If you are running locally a MUD TypeScript template version 2.2 or later, the URL for World Explorer is [http://localhost:13690](http://localhost:13690).
+
+<Tabs items={["Data Explorer", "Interact"]}>
+  <Tab>
+    <iframe
+      width="761"
+      height="523"
+      src="https://www.youtube.com/embed/87wuGFH9Pz0"
+      title="Using the MUD Data Explorer"
+      frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+      referrerpolicy="strict-origin-when-cross-origin"
+      allowfullscreen
+    ></iframe>
+  </Tab>
+  <Tab>
+    <iframe
+      width="761"
+      height="523"
+      src="https://www.youtube.com/embed/87wuGFH9Pz0?t=281https://www.youtube.com/embed/87wuGFH9Pz0?si=EsgcYGK9qk6J-Xvs&amp;start=281"
+      title="Using the MUD Data Explorer's Interact tab"
+      frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+      referrerpolicy="strict-origin-when-cross-origin"
+      allowfullscreen
+    ></iframe>
+  </Tab>
+</Tabs>
+
+### CLI options
+
+To use the explorer with different command-line options, use this process:
+
+1. In the mprocs screen, go down to the **explorer** process.
+
+1. Type `x` to stop the default explorer.
+
+1. In a different command-line window, go to `packages/contract`.
+
+1. Run the explorer using `pnpm explorer <options>`.
+
+The World Explorer accepts the following CLI options:
+
+| Option              | Description                                                         | Default value |
+| ------------------- | ------------------------------------------------------------------- | ------------- |
+| `--worldAddress`    | The address of the world to explore                                 | None          |
+| `--worldsFile`      | Path to a worlds configuration file (used to resolve world address) | worlds.json   |
+| `--indexerDatabase` | Path to your SQLite indexer database                                | indexer.db    |
+| `--chainId`         | The chain ID of the network                                         | 31337         |
+| `--port`            | The port on which to run the World Explorer                         | 13690         |
+| `--dev`             | Run in development mode                                             | false         |
+
+## Installation (pre-2.2 MUD)
+
+1. Install the Indexer and World Explorer packages in the project root.
+
+   ```sh copy
+   pnpm install @latticexyz/store-indexer@latest -w
+   pnpm install @latticexyz/explorer@latest -w
+   ```
```

Comment:
we might want to add a step here to set mud version to align all package versions 

```
pnpm mud set-version --tag latest
pnpm install
```